src/main/js/main_html5.js

```diff
@@ -1016,6 +1017,15 @@ require("../../../html5-common/js/utils/environment.js");
      * @param {object} event The event from the video
      */
     var raiseTimeUpdate = _.bind(function(event) {
+      // [PBW-6200] - MS Edge will not fire 'playing' or 'canplaythrough' events after seeking.
+      // If playback progresses and 'waitingEventRaised' hasn't been cleared, we fire a
+      // 'buffered' event in order to notify that the video has stopped buffering
```

Comment:
@aeng7 I need to confirm this, but I believe that the `timeupdate` event will not get fired until seeking is done. So if you seek again while the video is seeking, there won't be any `timeupdate`'s in between.

It does seem to me that `timeupdate` is the reliable way to detect that playback has resumed. The other options are `canplay` and `canplaythrough` which don't indicate that the video is actually playing (even though it usually is when these are fired), only that it can be played. Also, `play` and `playing` are supposed to be fired when playback starts after the video was stopped or paused, so some browsers interpret seeking as a pause, but Edge doesn't.